Assets/Localization/StringTables/Internal_RSC_en.asset

```diff
@@ -21941,7 +21941,7 @@ MonoBehaviour:
       yer pardon, but that''s such a stupid question, I don''t think I should haveta
       answer.[/record]
 
-      I reckin'' a little more wanderin'' around will make
+      I reckon a little more wanderin'' around will make
```

Comment:
Wasn't it "incorrect" on purpose to match the informal tone of the sentence?

Reply:
I left most instances of blunt speech/slang alone, but the apostrophe at the end shouldn't be there since it isn't a contraction. Misspellings like that are often used to phonetically indicate an accent, but "reckin" and "reckon" would be homophones, so it doesn't actually indicate anything about their speech in this case.
If it was a letter a character had written that'd be a different story (and I did leave such intentional instances alone, as in the Ripper quests).